macros/calibrateThrDac.py

```diff
@@ -494,7 +494,7 @@
             dict_ScurveMeanVsThrDac[vfat].GetPoint(thrDacIndexPairs[i][1],thrDacVal,scurveMean)
             scurveSigmaError = dict_ScurveSigmaVsThrDac[vfat].GetErrorY(thrDacIndexPairs[i][1])
             scurveMeanError = dict_ScurveMeanVsThrDac[vfat].GetErrorY(thrDacIndexPairs[i][1])
-            if scurveMean < 0.1:
+            if scurveMean < 0.1 or scurveMeanError/scurveMean < 0.001:
```

Comment:
I would put these values somewhere that they can be defined as either configuration parameters, or well-defined, documented constants

Reply:
they are moved to anaInfo.py now